.markdownlint.yaml

```diff
@@ -58,17 +58,17 @@ MD012:
 # MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md013.md
 MD013:
   # Number of characters
-  line_length: 100
+  line_length: 80
   # Number of characters for headings
   heading_line_length: 80
   # Number of characters for code blocks
-  code_block_line_length: 100
+  code_block_line_length: 80
```

Comment:
Psst: we should merge it while @elBoberido is sleeping, he is a defender of 110 characters per line ;)

Reply:
@elfenpiff This check is actually disabled for code blocks.  I just reset the value back to default. Or is 80 also a point of contention for text? 😆